concourse/enumerator.py

```diff
@@ -354,6 +354,10 @@ def _scan_repository_for_definitions(
             # handle inheritance
             definitions = merge_dicts(definitions, override_definitions)
 
+            for pipeline_name in tuple(definitions.keys()):
```

Comment:
~~why create a tuple from `definitions.keys()`?~~

instead of working around issues from manipulating iterated-over dict, why not create a new one, like so:

```
definitions = {
  name: definition for name, definition in definitions.items()
  if definition.get('render_pipeline', True)
}
```

wdyt? not 100% sure this variant is easier to read

Reply:
as an alternative, why not add a simple `continue` into  `_wrap_into_descriptors`?

Reply:
yep the latter sounds good to me 👍🏼